runtime/src/iree/hal/drivers/cuda/nccl_channel.c

```diff
@@ -6,10 +6,21 @@
 
 #include "iree/hal/drivers/cuda/nccl_channel.h"
 
+#include <iree/base/config.h>
```

Comment:
as with other file, all IREE includes are `"`, come below, and all base/ headers come in via `"iree/base/status.h"` already